snap/implicit.go

```diff
@@ -29,6 +29,7 @@ import (
 
 var implicitSlots = []string{
 	"alsa",
+	"autopilot-introspection",
 	"bluetooth-control",
 	"camera",
 	"dcdbas-control",
```

Comment:
Please add "autopilot_instrospection_test.go" and model it after network_control_test.go